webapp/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "Systems-Development-and-Frameworks",
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "serve": "vue-cli-service serve",
+    "build": "vue-cli-service build",
+    "lint": "vue-cli-service lint",
+    "test": "vue-cli-service test:unit --verbose"
+  },
+  "dependencies": {
+    "@babel/core": "^7.7.2",
+    "acorn": "^7.1.0",
+    "acorn-jsx": "^5.1.0",
+    "apollo-boost": "^0.4.4",
+    "babel-core": "^7.0.0-bridge.0",
+    "core-js": "^3.3.2",
+    "graphql": "^14.5.8",
+    "graphql-tag": "^2.10.1",
+    "vue": "^2.6.10",
+    "vue-apollo": "^3.0.0",
+    "vue-router": "^3.1.3"
```

Comment:
Same here probably. I believe, if you have a plain SPA, there is no real dependency because you are not running any node process.

Reply:
Whats SPA?

Reply:
Single-page-application. VueJS is a single-page-application framework. I assume you build a couple of JS, HTML and CSS files and then just serve it with an ordinary file server. In that case, there is no node process running on the server. Probably you can put everything into the `devDependencies` section.

Once we introduce NuxtJS we will have runtime dependencies because we're running a node process on the server.

Reply:
I don't really know which dependency goes where. Maybe you can make an Overview in an upcoming class?